crates/flux-middle/src/rustc/lowering.rs

```diff
@@ -663,6 +663,10 @@ fn lower_const<'tcx>(
         rustc_type_ir::ConstKind::Value(ty, ValTree::Leaf(scalar_int)) => {
             ConstKind::Value(lower_ty(tcx, ty)?, scalar_int)
         }
+        rustc_type_ir::ConstKind::Unevaluated(c) => {
+            let args = lower_generic_args(tcx, c.args)?;
```

Comment:
I doubt this will become a problem in the near future but we could raise unsupported if the list of args is not empty.